### PR TITLE
Wip 231 more tests

### PR DESCRIPTION
**Pull request by PaulineMauryL**

### CODE REVIEW

**Comment by PaulineMauryL** [comment]
## Coverage report


 <details><summary>Click to see where and how coverage changed</summary><table><thead>
  <tr><th>File</th><th>Statements</th><th>Missing</th><th>Coverage</th><th>Coverage<br>(new stmts)</th><th>Lines missing</th></tr>
</thead>
<tbody><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/private_dataset</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af">utils.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8af"></a></td><td><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-63f32a3e50743f125d1d2fffa0e91ccb35e89c98c53cdb69f5838fbb8398e8afR49-R54">49-54</a></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/tests</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c">test_api.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/utils</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7">utils.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-517d7a2783eedf30c47a796fd7e5f657d815318c06656e3b57bf6ad2a5dc39f7"></a></td><td></td></tbody>
<tfoot>
<tr>
<td><b>Project Total</b></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/79/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td>&nbsp;</td>
</tr>
</tfoot>
</table>

<sub>

This report was generated by [python-coverage-comment-action](https://github.com/py-cov-action/python-coverage-comment-action)

</sub>
</details>

File: .github/workflows/test_and_coverage_server.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ name: Server Integration Testing
 
 on: 
   pull_request:
-    branches: [ develop, master ]
```
Comment (damienbfs) [inline .github/workflows/test_and_coverage_server.yml:5]: We also need to push new docker images no? 
- So if we merge something in develop, we would tag it with the commit sha but remove the "latest" tag,
- and then if we merge into master we would rebuild the same image but tag it with the "latest" tag

Does that make sense?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 </h1><br>
 
 
-[![Coverage badge](https://raw.githubusercontent.com/dscc-admin-ch/lomas/python-coverage-comment-action-data/badge.svg)](https://htmlpreview.github.io/?https://github.com/dscc-admin-ch/lomas/blob/python-coverage-comment-action-data/htmlcov/index.html)
+[![Coverage badge](https://raw.githubusercontent.com/dscc-admin-ch/lomas/python-coverage-comment-action-data/badge.svg)](https://htmlpreview.github.io/?https://raw.githubusercontent.com/dscc-admin-ch/lomas/python-coverage-comment-action-data/htmlcov/index.html)
```
Comment (damienbfs) [inline README.md:6]: Nice!

**Review by damienbfs — APPROVED** [review]

File: .github/workflows/test_and_coverage_server.yml (hunk 1 above)
Comment (PaulineMauryL) [reply .github/workflows/test_and_coverage_server.yml:5]: Yes, that seems good to me

**Pull request merged**